week3/TaskOOP/src/com/JH/Account.java

```diff
@@ -0,0 +1,48 @@
+package com.JH;
+
+public class Account {
+
+    private int accountNumber;
```

Comment:
make account number as string instead . as if I have accounts more than the size of int and long it will lead to overflow.

Reply:
ok i'll do